manifest.json

```
@@ -6,7 +6,7 @@
         "en": "A little dashboard to analyse your electricity consumption data from Linky & weather",
         "fr": "Un petit tableau de bord pour analyser votre consomation d'électricité à partir des données d'un Linky et de la météo"
     },
-    "version": "0.5.3~ynh1",
+    "version": "0.5.5",
```

```suggestion
    "version": "0.5.5~ynh1",
```
We should leave `~ynh` after the version number